model/textparse/interface.go

```diff
@@ -80,7 +80,7 @@ type Parser interface {
 //
 // This function always returns a valid parser, but might additionally
 // return an error if the content type cannot be parsed.
-func New(b []byte, contentType string, parseClassicHistograms bool, st *labels.SymbolTable) (Parser, error) {
+func New(b []byte, contentType string, parseClassicHistograms, skipOMSeries bool, st *labels.SymbolTable) (Parser, error) {
```

Comment:
```suggestion
func New(b []byte, contentType string, parseClassicHistograms, skipOMCTSeries bool, st *labels.SymbolTable) (Parser, error) {
```

`skipOMSeries` sounds weird since we're not skipping all OM series, just CT.  `skiptCTSeries` would also be weird because only OpenMetrics 1.0 exposes CT with a "series format". What do you think about the suggestion? 😅

Reply:
Yes this sounds better!